packages/block-library/src/columns/index.js

```diff
@@ -62,8 +66,32 @@ export const settings = {
 
 	description: __( 'Add a block that displays content in multiple columns, then add whatever content blocks you’d like.' ),
 
-	supports: {
-		align: [ 'wide', 'full' ],
+	supports,
+
+	transforms: {
```

Comment:
Can we move the declaration of this transform the the `text-columns` block instead (`to` transform) that way when/if we remove the block, we don't forget the transform.